services/web/server/src/simcore_service_webserver/application_settings.py

```diff
@@ -174,6 +174,7 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
     WEBSERVER_REMOTE_DEBUG: bool = True
     WEBSERVER_SOCKETIO: bool = True
     WEBSERVER_TAGS: bool = True
+    WEBSERVER_FOLDERS: bool = True
```

Comment:
Maybe more clear: `WEBSERVER_USES_FOLDERS` or `WEBSERVER_FOLDERS_ENABLED`

Reply:
@mrnicegyu11 all plugin settings names are *normalized* to ``WEBSERVER_$(SERVICE_NAME)``. If the plugin has not settings it is ``bool``. This way to disable the plugin is always done in the same way: ``WEBSERVER_$(SERVICE_NAME)=0``